change/change-chatty-cloths-develop.json

```diff
@@ -0,0 +1,9 @@
+{
+"changes": [{
```

Comment:
One more question (sorry)--does this PR change the default way change files are written? I didn't see the new option being enabled for beachball itself, but it looks like this change file is using the new format.

Reply:
The update supports the multi-change format in the readChangefile, but the default in writeChangefile will still be regular way it is now.